concourse/pipelines/linux-image-build.jsonnet

```diff
@@ -239,6 +239,8 @@ local imgpublishjob = {
   trigger:: if tl.env == 'testing' then true
   else false,
 
+  citfilter:: '^(%s)$' % self.citsuites,
+  citsuites:: 'cvm|livemigrate|suspendresume|loadbalancer|guestagent|hostnamevalidation|imageboot|licensevalidation|network|security|hotattach|lssd|disk|packagevalidation|ssh|metadata|vmspec',
```

Comment:
The current default is kinda defined in a wrapper around common.imagetesttask on line 18. There should be a single definition of the defaults to avoid confusion - either drop the wrapper or define the defaults in another variable and reference it in both places.

Reply:
I dropped it in the line 18 in favor of next to the publish job definitions since that's where it's being used. If that placement doesn't make sense logically we can stick with having it defined globally vs just for the imgpublishjob where it's being used.